.github/workflows/dev_on_push_workflow_main.yml

```
@@ -202,6 +202,17 @@ jobs:
     secrets:
       GCP_EHP_SERVICE_ACCOUNT: ${{ secrets.GCP_EHP_SERVICE_ACCOUNT }}
       GCP_EHP_WORKLOAD_IDENTITY_PROVIDER: ${{ secrets.GCP_EHP_WORKLOAD_IDENTITY_PROVIDER }}
+  deploy-web-perf:
+    needs: [soft-deploy-testing, hard-deploy-android-testing, hard-deploy-ios-testing]
+    if: ${{ always() && contains(needs.*.result, 'success') && !contains(needs.*.result, 'failure') && github.ref == 'refs/heads/master'}}
```

pour ma culture perso, pourquoi ceci n'est pas suffisant ?

```suggestion
    if: ${{github.ref == 'refs/heads/master'}}
```

---

faudrait que je regarde le prq Dimitri avait fait ça, il y avait une raison avec le changed files il me semble mais je l'ai plus en tete